handbook/engineering/web/index.md

```diff
@@ -1,22 +1,83 @@
+  
+ℹ️ _Joel is the acting PM for this team until we are able to hire a dedicated PM for it. Once that happens, he will focus entirely on [Code Insights](../code-insights/index.md)._  
 
 ## Contact
 
-- #web channel or @web-team in Slack.
+- [#web](https://sourcegraph.slack.com/archives/CMT39K56Z) channel or @web-team in Slack.
 - [team/web](https://github.com/sourcegraph/sourcegraph/issues/new?labels=team/web) label and [@sourcegraph/web](https://github.com/orgs/sourcegraph/teams/web) team on GitHub.
 
 ## [Goals](goals.md)
 
 See [goals](goals.md)
 
+## Growth plan
+
+ℹ️ _Updated TODO_
+
+We are actively hiring [frontend engineers](../hiring/software-engineer-frontend.md) to grow the Web team. 
+When this team team gets big enough, we expect to split into two teams **Frontend platform and code browsing** and **Extensions and Integrations**
+
+ℹ️ _As of TODO, [Code Insights](../code-insights/index.md) moved out of the Web Team into it's own team, lead by Felix Becker (Interim EM) and Joel Kwartler (PM)._
+
+### Frontend platform and code browsing
+
+This team is responsible for:
+
+1. Defining how we do web development at Sourcegraph and ensuring that our product engineering teams have the tools and components they need to quickly build high quality user experiences everywhere we use web technologies (for example: web application, browser extensions).
+1. The core user experience of the Sourcegraph product
+
+The team's areas of responsibility include:
+
+- Frontend platform
+   - Creating and maintaining a standard Sourcegraph UI component library.
+   - Sourcegraph web tech stack, tools, patterns, documentation, education (e.g. webpack, TypeScript, React, RxJS).
+   - Documentation and training material to enable product teams and new hires to quickly learn how we do web development at Sourcegraph.
+   - Define and maintain how we test frontend code.
+   - Ensuring an efficiency and reliabile frontend CI pipeline.
+- Code browsing
+   - Application navigation and information hierarchy
+   - Syntax highlighting and file viewing
+   - Repository landing pages and file directory pages
+   - Basic code host-like features (e.g. viewing diffs, commits, history)
+   - Sourcegraph application homepage
+   - Accessibility support
+
+### Extensions and integrations
+
+This team is responsible for: 
+
+1. Bringing the value of Sourcegraph to other developer tools
+1. Bringing the value of other developer tools into Sourcegraph
+
+The team's areas of responsibility include:
+
+- Sourcegraph extensions
+   - Providing a Sourcegraph extension API that enables developers to bring data from their favorite developer tools into their Sourcegraph workflow.
+   - Building useful Sourcegraph extensions on top of the Sourcegraph extension API (for example: Codecov, Datadog, Sentry, Lightstep)
+   - Documentation and tutorials that enable third party developers to create extensions that are actively used.
```

Comment:
I would add or modify a bullet here to also include the extensions registry itself. Like perhaps 
`Building and maintaining the extensions registry and discovery paths to create a community around Sourcegraph extensions`, which is on the long term roadmap (at least if I have any say) for extensions.